src/OrchardCore.Themes/TheTheme/Views/MenuItemLink-LinkMenuItem.cshtml

```diff
@@ -1,20 +1,16 @@
 @using OrchardCore.ContentManagement
-
 @{
     ContentItem contentItem = Model.ContentItem;
     var linkMenuItemPart = contentItem.Content["LinkMenuItemPart"];
```

Comment:
this should use `.As` (I know you didn't write it. but if you're going to fix things ;)

Reply:
Done.